src/components/ga4/EnhancedEcommerce/add-to-cart.module.css

```diff
@@ -0,0 +1,18 @@
+.addToCart {
+  display: flex;
+  flex-direction: row;
+  color: var(--text-color-inverted);
+  background-color: var(--primary);
+  align-self: flex-end;
+  padding: var(--space-sm) var(--space-xl);
+  border-radius: var(--radius-md);
+  font-weight: var(--bold);
+  align-items: center;
+  height: var(--size-input);
+  justify-content: center;
+  transition: var(--transition);
+}
+
+.addToCart:hover {
+  box-shadow: var(--shadow);
+}
```

Comment:
can you run:
```
yarn prettier && yarn lint -- --fix
```
to clean up these newlines and such? If it doesn't do anything don't worry about it. I'm not positive I set formatting to work with css files.